### PR TITLE
sar: add page

### LINKED ISSUE #1529: page request: sar
The system activity reporter `sar` is not documented. I want to give it a try :)

### PR DESCRIPTION
**Pull request by mfrw**
Fixes #1529.
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/linux/sar.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# sar
+
+> Monitor performance of various Linux subsystems.
+
+- Report I/O and Transfer rate every 1 second:
```
Comment (sbrl) [inline pages/linux/sar.md]: Don't think `Transfer` needs to have a capital letter at the beginning.

File: pages/linux/sar.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+`sar -b 1`
```
Comment (sbrl) [inline pages/linux/sar.md]: There aren't any long forms of the options?

File: pages/linux/sar.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+- Report 10 network statistics every 2 seconds for network devices:
+
+`sar -n DEV 2 10`
+
+- Report CPU utilization every 2 seconds:
+
+`sar -u ALL 2`
+
+- Report 20 memory utilization statistics every 1 second:
```
Comment (sbrl) [inline pages/linux/sar.md]: This feels a bit awkward. Perhaps we could reword this to something like ...`statistics, one per second`?

File: pages/linux/sar.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+`sar -r 1 20`
+
+- Report queue length and load averages:
+
+`sar -q 1 1`
+
+- Report paging statistics every 1 second:
```
Comment (sbrl) [inline pages/linux/sar.md]: Perhaps we could change this example to every 5 seconds to change things up? :smiley_cat:

File: pages/linux/sar.md (hunk 1 above)
Comment (agnivade) [inline pages/linux/sar.md]: What transfer rate are we talking about ?

File: pages/linux/sar.md (hunk 2 above)
Comment (agnivade) [inline pages/linux/sar.md]: Please wrap 1 in tokens.

File: pages/linux/sar.md (hunk 1 above)
Comment (mfrw) [reply pages/linux/sar.md]: sure, acutally the manpage had I/O and Transfer in upper-case, so i did the same.
anyways changed

File: pages/linux/sar.md (hunk 2 above)
Comment (mfrw) [reply pages/linux/sar.md]: unfortunately not :(

File: pages/linux/sar.md (hunk 3 above)
Comment (mfrw) [reply pages/linux/sar.md]: done..

File: pages/linux/sar.md (hunk 4 above)
Comment (mfrw) [reply pages/linux/sar.md]: done

File: pages/linux/sar.md (hunk 1 above)
Comment (mfrw) [reply pages/linux/sar.md]: physical devices, added to the description

File: pages/linux/sar.md (hunk 2 above)
Comment (mfrw) [reply pages/linux/sar.md]: sorry for missing it ...
fixed

File: pages/linux/sar.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+- Report I/O and transfer rate issued to physical devices, every second (press CTRL+C to quit):
+
+`sar -b {{1}}`
+
+- Report a total of 10 network device statistics, one per 2 seconds:
+
+`sar -n DEV {{2}} {{10}}`
+
+- Report CPU utilization every 2 seconds (press CTRL+C to quit):
```
Comment (agnivade) [inline pages/linux/sar.md]: No need to mention 'press ctrl-c to quit' after the first time. It's obvious.

File: pages/linux/sar.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+`sar -u ALL {{2}}`
+
+- Report a total of 20 memory utilization statistics,  one per second:
```
Comment (agnivade) [inline pages/linux/sar.md]: double space between "," and "one"

File: pages/linux/sar.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+`sar -r ALL {{1}} {{20}}`
+
+- Report the run queue length and load averages:
+
+`sar -q {{1}} {{1}}`
```
Comment (agnivade) [inline pages/linux/sar.md]: Does the first 1 mean its once per second, should we mention that if that is the case ?

File: pages/linux/sar.md (hunk 3 above)
Comment (agnivade) [reply pages/linux/sar.md]: Should it be "once per second" ? I think that sounds better and more natural.

File: pages/linux/sar.md (hunk 5 above)
Comment (mfrw) [reply pages/linux/sar.md]: sure

File: pages/linux/sar.md (hunk 6 above)
Comment (mfrw) [reply pages/linux/sar.md]: my bad :(

File: pages/linux/sar.md (hunk 7 above)
Comment (mfrw) [reply pages/linux/sar.md]: almost every command has the same structure, the first arg is the refresh granularity, and the second (if any) is the total number of stats to print. 
If the second arg is not provided, it assumes infinity.

So, I think this is obvious, and we should not write it explicitly to confuse people

Comment (agnivade) [reply pages/linux/sar.md]: I got a bit confused because you mention 2 things - queue length and load average, and then there are 2 params. Its not very obvious whether the two "1"s are for granularity and no. of stats, or the granularity for queue length and load svg.

That's why I wanted to be extra clear in case there's any confusion. What do you think ?

File: pages/linux/sar.md (hunk 3 above)
Comment (agnivade) [reply pages/linux/sar.md]: On further thought, I think it should be changed to "once every second". Because the other examples are of the form "every 2 seconds", "every 5 seconds" and then you have "one per second", "one per 2 seconds".

Either we use "per second" everywhere or "every second" everywhere. But mixing the 2 does not sound right to me.

**Comment by agnivade** [comment]
Hey @mfrw - left a couple of comments. 

Sorry for the back and forth. Our review process might feel a bit tedious. Thank you for the patience !

File: pages/linux/sar.md (hunk 7 above)
Comment (mfrw) [reply pages/linux/sar.md]: I think lets just skip the second param, if anyone wants, it is present in the man page.
This would make it less confusing.

**Comment by mfrw** [comment]
@agnivade  your review process is amazing. I am getting to learn a lot of things. I am new to open-source.
Thanks for your patience :)

**Comment by sbrl** [comment]
Thanks for the hard work, @mfrw! :smiley_cat:
